azkaban-web-server/src/main/java/cloudflow/services/ProjectServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package cloudflow.services;
+
+import azkaban.user.User;
+import cloudflow.daos.ProjectDao;
+import cloudflow.error.CloudFlowNotFoundException;
+import cloudflow.models.Project;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class ProjectServiceImpl implements ProjectService {
+
+  private final ProjectDao projectDao;
+  private static final Logger log = LoggerFactory.getLogger(ProjectServiceImpl.class);
+
+  @Inject
+  public ProjectServiceImpl (ProjectDao projectDao) {
+    this.projectDao = projectDao;
+  }
+
+  @Override
+  public String createProject(Project project, User user) {
+    return this.projectDao.create(project, user);
```

Comment:
Consider capturing DAO layer exceptions and creating the corresponding service layer exceptions, to abstract away the persistence layer from servlets.

Reply:
Will improve exception handling in follow-up change since other changes are waiting on this.